packages/optimizer/src/lib.rs

```diff
@@ -656,97 +309,116 @@ fn minimize(
     log::info!("-------------------------------------");
     log::info!("minimize");
 
-    // (10,000 steps / 100ms) * (10 ms / s) (???) = 100k steps/s (on this simple problem (just `sameCenter` or just `contains`, with no line search, and not sure about mem use)
-    // this is just a factor of 5 slowdown over the compiled energy function
-
     let mut xs = xs0.to_vec(); // Don't use xs
     let mut fxs = 0.;
     let mut gradfxs = vec![0.; xs0.len()];
     let mut gradient_preconditioned = gradfxs.clone();
     let mut norm_gradfxs = 0.;
-    let mut i = 0;
-    let mut t = 0.0001; // NOTE: This const setting will not necessarily work well for a given opt problem.
-    let mut failed = false;
 
-    let mut new_lbfgs_info = lbfgs_info;
+    let cfg = lbfgs::Config {
+        m: 17,
+        armijo: 0.001,
+        wolfe: 0.9,
+        min_interval: 1e-9,
```

Comment:
This was changed from `1e-10` to `1e-9`, for example. Why?

Reply:
Previously it was `10e-10`, which is the same as `1e-9`.